docs/_assets/codepen.js

```diff
@@ -102,6 +102,15 @@ CodeBlockCodePen.prototype.clickHandler = function(form, pre) {
 
   return function() {
 
+    // Track codepen button clicks
+    if (typeof ga !== 'undefined') {
+      ga('send', {
+        hitType: 'event',
+        eventCategory: 'codepen',
+        eventAction: 'click'
```

Comment:
Wait at least I can easily track the current URL it will tell us for which component is the button that;s already a good start :D